src/main.zig

```diff
@@ -2475,6 +2475,8 @@ fn buildOutputType(
                         fatal("unable to parse /version '{s}': {s}", .{ arg, @errorName(err) });
                     };
                     have_version = true;
+                } else if (mem.eql(u8, arg, "-v")) {
+                    try std.io.getStdOut().writeAll("zig ld " ++ build_options.version ++ "\n");
                 } else if (mem.eql(u8, arg, "--version")) {
```

Comment:
While you're here, could you add a branch that handles `-V` as well, which just does `warn("ignoring request for supported emulations: unimplemented", .{});`?

Reply:
> While you're here, could you add a branch that handles `-V` as well, which just does `warn("ignoring request for supported emulations: unimplemented", .{});`? It should likewise not exit.

I'll have a look at this one in the meantime